src/homework6/Chamomile.java

```diff
@@ -0,0 +1,14 @@
+package homework6;
+
+public class Chamomile extends Flower {
+    private int price = 70;
```

Comment:
better add constructor with param to parent class instead of writing the same field and method in every subclass